pkg/storage/snapshot/source.go

```diff
@@ -222,6 +238,35 @@ func (s *vmSnapshotSource) Unlock() (bool, error) {
 	return true, nil
 }
 
+func (s *vmSnapshotSource) verifyVolumesBound() (bool, error) {
+	pvcs, err := s.PersistentVolumeClaims()
+	if err != nil {
+		if storageutils.IsErrNoBackendPVC(err) {
+			// No backend pvc when we should have one, lets wait
+			// TODO: Improve this error handling
+			return false, nil
+		}
+		return false, err
+	}
+
+	for _, pvcName := range pvcs {
+		obj, exists, err := s.controller.PVCInformer.GetStore().GetByKey(cacheKeyFunc(s.vm.Namespace, pvcName))
+		if err != nil {
+			return false, err
+		}
+		if !exists {
+			return false, k8serrors.NewNotFound(appsv1.Resource("persistentvolumeclaim"), pvcName)
+		}
+
+		pvc := obj.(*corev1.PersistentVolumeClaim).DeepCopy()
+		if pvc.Spec.VolumeName == "" {
```

Comment:
Don't they have a status.phase == v1.Bound?

Reply:
I believe this is the best way to make sure the PVC is bound. Thats what we do in mot recent codes for example in the populators code.

Reply:
I think we changed it recently, can you point me to it

Reply:
You are right, thanks I will change